app/models/bus_stop_field.rb

```diff
@@ -0,0 +1,13 @@
+class BusStopField < ApplicationRecord
+  belongs_to :bus_stop
+  belongs_to :field, foreign_key: :field_name
+  
+  validate :check_choice_included
+  
+  private
+    def check_choice_included
+      if field.multiple_choice? && !field.choices.include?(value)
+        errors.add :value, 'is not a valid choice for this field'
```

Comment:
Is the purpose of this to ensure the response matches with Hastus?

Reply:
Making sure that the value is one of the possible choices.

Reply:
Right. I guess this is kind of a moot point since I don't disagree with this validation. Does Hastus really care what string we send it? That's my real question. You would probably have to be selecting a different answer intentionally to get something outside of the options.

Reply:
We are considering how to implement a Hastus-side validation that a choice is in a particular list.